### ISSUE TITLE
CLI Crashed!

### ISSUE THREAD

**Issue by santhoshnv** [angular/angular-cli#15050]
# 🐞 Bug report

### Command (mark with an `x`)

```
- [ ] new
- [x] build
- [ ] serve
- [ ] test
- [ ] e2e
- [ ] generate
- [ ] add
- [ ] update
- [ ] lint
- [ ] xi18n
- [ ] run
- [ ] config
- [ ] help
- [ ] version
- [ ] doc
```

### Is this a regression?

 Yes, the previous version in which this bug was not present was: ....

### Description
 A clear and concise description of the problem...
I can't able to compile the project

## 🔬 Minimal Reproduction

I tried to run the project using the following command, It's compiled successfully for the first 2 times. But the next time it got failed

compilation code - `node --max-old-space-size=10096 ./node_modules/@angular/cli/bin/ng serve --configuration dev`

## 🔥 Exception or Error
<pre><code>

Assertion `!(n > 0) || (ret != nullptr)' failed.
</code></pre>

## 🌍 Your Environment
<pre><code>

    "@angular/animations": "^7.2.1",
    "@angular/common": "^7.2.1",
    "@angular/core": "^7.2.1",
    "@angular/forms": "^7.2.1",
    "@angular/http": "^7.2.1",
    "@angular/platform-browser": "^7.2.1",
    "@angular/platform-browser-dynamic": "^7.2.1",
    "@angular/platform-server": "^7.2.1",
    "@angular/router": "^7.2.1",
    
   Node version  - 10.15.3
</code></pre>

**Comment by elvisbegovic** [comment]
@santhoshnv Hi again, this is really what is printed when you run `ng version` under your project? 
It seems it misses a lot of other version infos. 
Can paste whole content of `ng version`?
Is it possible to make repro, share your projet if it is not a fresh new project

**Comment by alan-agius4** [comment]
Hi, this seems to be related to a Node bug https://github.com/nodejs/node/issues/20297 which was supposed to be addressed in 10.5.0.

I suggest a couple of things here.
1) Update to the latest version of @angular/cli 
2) Update your Node JS version.

If that doesn't solve the issue, kindly fill a new issue with a minimal reproduction or the exact steps to reproduce the error.

Thanks.

**Comment by santhoshnv** [comment]
This my cli information

![image](https://user-images.githubusercontent.com/19868656/61123580-5b119380-a4c2-11e9-9c89-d73c42686b6b.png)
